templates/footer.hbs

```diff
@@ -0,0 +1,24 @@
+<footer class="container--color">
+    <div class="footer">
+        <div class="footer__logo">
+            <a href="/index.html"><img src="/images/transformasion-logo-large.png" alt=""></a>
```

Comment:
Set an alt text for the logo. Something like `alt="Transformasion"`

Reply:
Done